Postman/dex.postman_collection.json

```diff
@@ -881,6 +881,46 @@
 										}
 									},
 									"response": []
+								},
+								{
+									"name": "Project-Upload-Image-Administrator",
+									"request": {
+										"method": "POST",
+										"header": [
+											{
+												"key": "IdentityId",
+												"value": "{{administratorUserIdentityId}}",
+												"type": "text"
+											}
+										],
+										"body": {
+											"mode": "formdata",
+											"formdata": [
+												{
+													"key": "File",
+													"type": "file",
+													"src": "/C:/Users/Niray/Desktop/angular-226066.png"
```

Comment:
Maybe set up one file included with the postman tests and configured in env variables so testing can always be done without manually having to configure the body with a file present on current tester host.

Reply:
Added variables, unfortunately only works with CLI. Added .bat file and for now didn't yet make the right tests for file related requests. We should add them when postman tests are integrated with GitHub actions.